src/main/java/frc/robot/scoreStates/ScoreState.java

```diff
@@ -2,16 +2,21 @@
 
 import edu.wpi.first.wpilibj.DriverStation;
 import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.Commands;
 
 public interface ScoreState {
     default boolean isRed() {
         var alliance = DriverStation.getAlliance();
         return alliance.isPresent() && alliance.get() == DriverStation.Alliance.Red;
     }
 
+    default Command initializeCommand() {
+        return Commands.none();
+    }
+
     Command driveToClosestOptimalPoint();
 
-    Command stateInitialize();
+    Command initializeSubsystems();
 
     Command score();
```

Comment:
The order of these should match the order of execution.

Reply:
af1227d

Reply:
And what about changing the order in the implementing classes, too? 🙃

Reply:
b3880c2